Backtracking/tests/MinMax.test.js

```diff
@@ -0,0 +1,29 @@
+import { MinMax } from '../MinMax'
+
+describe('MinMax', () => {
+  it('should return 65 for MinMax([90, 23, 6, 33, 21, 65, 123, 34423],0,0,true)', () => {
```

Comment:
This test is practically equivalent to the below test. You're also duplicating the contents in the description. Please see the [TypeScript guide for writing good tests](https://github.com/TheAlgorithms/TypeScript/blob/master/CONTRIBUTING.md#writing-good-tests).